pkg/parser/additional/additional.go

```diff
@@ -0,0 +1,199 @@
+package additional
```

Comment:
Maybe we should think of a better name to put the utility functions.

`certificate_utils.go` could be a candidate.